app/views/shared/_footer.html.erb

```diff
@@ -29,14 +28,13 @@
       <div class="col-md-4 col-xs-5">
         <div class="text-right">
           <p>
-            <img class='social-logo' src='<%= image_url('insta.png')%>' alt="Instagram">
-            <img class='social-logo' src='<%= image_url('facebook.png')%>' alt="Facebook">
-            <img class='social-logo' src='<%= image_url('twitter.jpg')%>' alt="Twitter">
+            <a href="https://twitter.com/oregondigcol"><img class='social-logo' src='<%= image_url('twitter.jpg')%>' alt="Twitter"></a>
+            <a class="footer-link-text" href="https://twitter.com/oregondigcol">@oregondigcol</a>
           </p>
           <p class='deployed-version'>
             Oregon Digital <%= "#{OregonDigital::VERSION}-#{ENV.fetch('DEPLOYED_VERSION', 'development')}" %>
           </p>
-          <p>Powered by <a class="footer-link-text" href="samvera.org">Samvera</a> Hyrax <%= Hyrax::VERSION %></p>
+          <p>Powered by <a class="footer-link-text" href="http://www.samvera.org">Samvera</a> Hyrax <%= Hyrax::VERSION %></p>
```

Comment:
https all the things! 😄

Reply:
Ryan wick said no https here `Fix Samvera link (no https)`

Reply:
I read that as that's what's wrong with the link
`http://www.samvera.org` redirects to `https://www.samvera.org` anyways, it's just nicer for everyone involved.

Reply:
Heh. Yeah. I think i just interpreted it oddly. I fixed it.